core/ecschema-metadata/tsconfig.json

```diff
@@ -1,9 +1,10 @@
 {
   "extends": "./node_modules/@itwin/build-tools/tsconfig-base.json",
-  "compilerOptions": {
-    "outDir": "./lib"
-  },
   "include": [
-    "src/**/*.ts"
+    "./src/**/*.ts"
+  ],
+  "exclude": [
```

Comment:
Any reason the excludes are needed here?

Reply:
reverting